src/app/zap-templates/zcl/data-model/chip/scene.xml

```diff
@@ -15,7 +15,12 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 <configurator>
-  <bitmap name="ScenesCopyMode" type="bitmap8">
+  <bitmap name="NameSupportBitmap" type="BITMAP8">
```

Comment:
Doesn't the NameSupport attribute need to be of this type, per spec?  And then you would not need the `Raw()` thing..

This is the "changes requested" bit.

Reply:
Indeed, type updated.